src/main/com/fulcrologic/rad/report_options.cljc

```diff
@@ -359,5 +359,23 @@
   :com.fulcrologic.rad.report/BodyItem)
 
 (def query-inclusions
-  "A vector of things to add to the top-level report's query. "
+  "A vector of things to add to the top-level report's query. 
+   
+   Example:
+  
+   Provided you have a `Person` component and have executed
+   `(df/load! [:person/id 123] Person)` you can query it:
+  
+   ```
+   ro/query-inclusions [{[:person/id 123] (comp/get-query Person)}]
```

Comment:
I'd prefer a much simpler example. No one is going to hard-code an entity ID.  Typically for a report these will be things like some ID that you don't want to display, but you need on the row. Say you want to include a row action where you just to the user's most recent comment, you might include `:user/most-recent-comment-id`.

If you actually want that to appear as a column, though, you'd just generate a "virtual" attribute (not on a schema) and add that as a column, at which point the column formatter could build the text as a link or button.

Reply:
I understand. Simplified.